features/compose_message.feature

```diff
@@ -4,28 +4,25 @@ Feature: Compose messages
   I should be able to compose a message
 
   Background:
-    Given the following user exist
-      | name      | password          | email         |
-      | bob       | bob12345          | bob@bob.com   |
-      | lisa      | lisa12345         | lisa@lisa.com |
-
+    Given the user "bob" exist
+    And the user "lisa" exist
     And I am logged in as "bob"
 
   Scenario: User doesn't enter in required subject field
-    Given I am on landing page
-    And I click on the "Inbox" button
-    And I click on the "Compose" button
-    And I select "lisa" from the recipient list
-    And I enter "good" in "Body"
-    And I click on the "Send Message" button
+    Given I visit "landing" page
+    And I click "Inbox" button
+    And I click "Compose" button
+    And I select "lisa" from the "Recipient"
```

Comment:
I would rephrase this to `I select "lisa" from "Recipient"`. Also, plaese use capital letters in names `lisa` -> `Lisa`